components/bl-animatedBox-component/src/index.js

```diff
@@ -0,0 +1,92 @@
+import { useCallback, useEffect, useMemo, useState } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const INFINITE = 'infinite';
+const DEFAULT_ZERO = 0;
+const DEFAULT_ONE = '1s';
+
+export default function AnimatedBoxComponent({ component, elRef, eventHandlers, pods }) {
+  const { style, classList, display, type, duration, delay, loop, autoStart } = component;
+  const { onMouseOver, onMouseOut, onClick, onAnimationStart, onCycleComplete, onAnimationEnd } = eventHandlers;
+
+  const [isAnimating, setIsAnimating] = useState(false);
+  const [hasStarted, setHasStarted] = useState(autoStart);
```

Comment:
Could you describe if few words why we need 'hasStarted' and 'isAnimating' in the same time?

Reply:
For me it looks like we could have something like below, and it will still works

```suggestion
  const [isAnimating, setIsAnimating] = useState(autoStart);
```